packages/tldraw/src/lib/shapes/text/toolStates/Pointing.ts

```diff
@@ -14,13 +14,24 @@ export class Pointing extends StateNode {
 	shape?: TLTextShape
 
 	markId = ''
+	originScreenPoint = null as null | Vec
+
+	override onEnter = () => {
+		this.originScreenPoint = this.editor.inputs.currentScreenPoint.clone()
+	}
 
 	override onExit = () => {
 		this.editor.setHintingShapes([])
 	}
 
 	override onPointerMove: TLEventHandlers['onPointerMove'] = (info) => {
-		if (this.editor.inputs.isDragging) {
+		// Don't create a fixed width shape unless the the drag is a little larger,
+		// otherwise you get a vertical column of single characters if you accidentally
+		// drag a bit unintentionally.
+		if (
+			this.editor.inputs.isDragging &&
+			!Vec.DistMin(this.editor.inputs.currentScreenPoint, this.originScreenPoint!, 32)
+		) {
```

Comment:
```suggestion
		// Don't create a fixed width shape unless the the drag is a little larger,
		// otherwise you get a vertical column of single characters if you accidentally
		// drag a bit unintentionally.
		const { editor } = this
		const { isPointing, originPagePoint, currentPagePoint } = editor.inputs
		if (
			isPointing &&
			Vec.Dist2(originPagePoint, currentPagePoint) >
				((editor.getInstanceState().isCoarsePointer
					? editor.options.coarseDragDistanceSquared
					: editor.options.dragDistanceSquared) *
					2) / // double the necessary drag distance for text shapes
					editor.getZoomLevel()
		) {
```

Let's copy the dragging code from the editor and add a multiplier (2x here). We shouldn't need to clone the origin screen point, that will be set just before entering and remain stable until pointer up.

Reply:
Might be easier for me to push a commit that does this,since there are a few other places I'd change.